odoo/addons/base/tests/test_expression.py

```diff
@@ -998,16 +998,18 @@ def test_accent(self):
         self.assertEqual(helen, self._search(Model, [('name', 'ilike', 'hélène')]))
         self.assertEqual(helen, self._search(Model, [('name', '=ilike', 'Hel%')]))
         self.assertEqual(helen, self._search(Model, [('name', '=ilike', 'hél%')]))
-        self.assertNotIn(helen, self._search(Model, [('name', 'not ilike', 'Helene')]))
-        self.assertNotIn(helen, self._search(Model, [('name', 'not ilike', 'hélène')]))
```

Comment:
Why removing test on `not ilike` specifically? Just add yours ?
Same below

Reply:
That doesn't change the fact that `not ilike` is not test anymore :thinking:

Reply:
Tests added back in the previous section.